example/vgg16_example_in_c.c

```diff
@@ -42,11 +44,22 @@ int main() {
     ERROR_CHECK(menoh_build_variable_profile_table(vpt_builder, model_data,
                                                    &variable_profile_table));
 
+    /*
     int32_t softmax_out_dims[2];
     ERROR_CHECK(menoh_variable_profile_table_get_dims_at(
       variable_profile_table, softmax_out_name, 0, &softmax_out_dims[0]));
     ERROR_CHECK(menoh_variable_profile_table_get_dims_at(
       variable_profile_table, softmax_out_name, 1, &softmax_out_dims[1]));
+    */
```

Comment:
Is it necessary to keep the old code as comment?

Reply:
I won't deprecate `get_dims_size()` and `get_dims_at()`. They are usable in some situations so I would like to show their usage as well. Do you have any idea?

Reply:
I see. That makes sense.